active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+
+### ScrollingView, additional events
+
+| **Member** | **Description**                                                            |
+|------------|----------------------------------------------------------------------------|
+| ExtentChanged | Raised when any of the two ExtentWidth and ExtentHeight properties changed. |
+| State | Gets the current interaction state of the control | Idle |
+| ScrollAnimationStarting | Raised when a ScrollTo or ScrollBy call triggers an animation. Allows customization of that animation |
+| ScrollCompleted | Raised at the end of a ScrollTo, ScrollBy, or ScrollFrom asynchronous operation. Provides the original ScrollingScrollInfo struct |
+| ZoomAnimationStarting | Raised when a ZoomTo or ZoomBy call triggers an animation. Allows customization of that animation. |
+| ZoomCompleted | Raised at the end of a ZoomTo, ZoomBy, or ZoomFrom asynchronous operation. Provides the original ScrollingZoomInfo struct. |
+| BringingIntoView | Raised at the beginning of a bring-into-view-request participation. Allows customization of that participation.
+
+
+### ScrollingView, additional methods
+
+| **Member** | **Description**                                                            |
+|------------|----------------------------------------------------------------------------|
+| ScrollTo(double horizontalOffset, double verticalOffset) | Asynchronously scrolls to specified offsets. Allows animation, respects snap points. |
+| ScrollTo(double horizontalOffset, double verticalOffset, ScrollingScrollOptions options) |Asynchronously scrolls to specified offsets with optional animation, with optional snap points respecting.
+| ScrollBy(double horizontalOffsetDelta, double verticalOffsetDelta) | Asynchronously scrolls by the provided delta amount. Allows animation, respects snap points.
+| ScrollBy(double horizontalOffsetDelta, double verticalOffsetDelta, ScrollingScrollOptions options) | Asynchronously scrolls by the provided delta amount with optional animation, with optional snap points respecting.
+| ZoomTo(float zoomFactor,IReference<Vector2> centerPoint) | Asynchronously zooms to specified zoom factor. Allows animation.
+| ZoomTo(float zoomFactor, IReference<Vector2> centerPoint, ScrollingZoomOptions options) | Asynchronously zooms to specified offsets with optional animation.
+| ZoomBy(float zoomFactorDelta, IReference<Vector2> centerPoint) | Async
```

Comment:
Why `0xFF` instead of `0xFFFF` (ManipulationModes) or `0xFFFFFFFF` like `DisabledFormattingAccelerators`?

Reply:
Design guidelines say to use `0xFFFFFFFF`

Reply:
Recommend: 0xFFFFFFFF